src/arrayMethodJoin.js

```diff
@@ -5,8 +5,27 @@
  *
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function() {
-    // write code here
+  [].__proto__.join2 = function(token = ',') {
+    let str = '';
+    let addToken = String(token);
```

Comment:
addToket - is function name. To avoid using let, you may assign to const with ternary operator